postgres/query.js

```diff
@@ -0,0 +1,19 @@
+const dbConfig = require('./psqlconfig.js');
+
+dbConfig.client.connect();
+
+const getListingById = (id) => {
+  const queryString = `SELECT * FROM listings WHERE id = ${id};`;
```

Comment:
Maybe minor -- but I wonder if there's a way you could revise this to prevent an SQL injection attack on your db?

Reply:
Thanks for the feedback @rogen-code ! Are you referring to the fact I pass the req.params.id into the db function and then added it as part of the string, instead of using args? Would you mind expanding on what you think I could do to prevent a SQL injection attack, I haven't looked into that much yet. Thanks 🙂